test/jdk/sun/tools/jinfo/JInfoTest.java

```diff
@@ -38,6 +38,7 @@
  * @test
  * @summary Unit test for jinfo utility
  *
+ * @requires vm.flagless
```

Comment:
I'm not so sure about this one. There are a number of things that change in the output based on JVM args. It would seem we would want to test this.

Reply:
updated the test